ui/app/components/selectable-card-container-graph.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+/**
+ * @module MetricsSelectableCardContainerGraph
```

Comment:
nit: looks like you updated the name of the file, so the  name of the component would instead be SelectableCardContainerGraph

Reply:
This is generated data that comes from when you use the ember CLI to generate a component.  It also added the Metrics in front of it because it's in that folder, but you're right it's confusing.  I'll remove this.